tests/acceptance/test_project_settings_sampling.py

```diff
@@ -290,7 +351,34 @@ def test_remove_specific_rule(self):
                     event=audit_log.get_event_id("PROJECT_EDIT"),
                 )
 
-    def test_activate_uniform_rule(self):
+    @mock.patch("sentry.api.endpoints.project_dynamic_sampling.raw_snql_query")
+    @mock.patch(
+        "sentry.api.endpoints.project_dynamic_sampling.discover.query",
+    )
+    def test_activate_uniform_rule(self, mock_query, mock_querybuilder):
+        mock_query.return_value = mocked_discover_query(self.project.slug)
+        mock_querybuilder.side_effect = [
+            {
+                "data": [
+                    {
+                        "trace": "6503ee33b7bc43aead1facaa625a5dba",
+                        "id": "6ddc83ee612b4e89b95b5278c8fd188f",
+                        "random_number() AS random_number": 4255299100,
+                        "is_root": 1,
+                    }
+                ]
+            },
+            {
+                "data": [
+                    {
+                        "project": self.project.slug,
+                        "project_id": self.project.id,
+                        "count": 2,
```

Comment:
jFYI if count is 2, previous mock should return 2 transactions.

Reply:
Thanks for the feedback, but I think this is not relevant to this specific test, but I can update it in a follow-up.

Reply:
correct, that's why i approved it already.